### PR TITLE
Pal 821 jvm e2e test

### PR DESCRIPTION
**Pull request by dev930018**

### CODE REVIEW

File: example-runner/src/main/resources/application-static.yaml [hunk 1]
```diff
@@ -14,10 +14,11 @@
...
+  directory: "../Palisade-examples/resources/data/"
+  filename: "../Palisade-examples/resources/data/employee_file0.avro"
 
 web:
   client:
     palisade-service: "localhost:8084"
-    filtered-resource-service: "localhost:8092"
\ No newline at end of file
+    filtered-resource-service: "localhost:8092"
+    data-service: "localhost:8082"
```
Comment (m78233) [inline example-runner/src/main/resources/application-static.yaml:24]:
```suggestion
    data-service: "localhost:8082"
    
```

File: example-library/src/main/resources/application-example.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@
 # Not interpreted by any java code, simply used by reference in the population.resources section
 default:
   rootId: ../Palisade-examples/resources
-  connectionDetail: data-service
+  connectionDetail: localhost:8082
```
Comment (m78233) [inline example-library/src/main/resources/application-example.yaml:19]: Just a question.  We use data-service in other aspects of the code, but here it is changed a port on localhost.  Why is this different?

File: deployment/local-jvm/example-runner/runLocalJVMExample.sh [hunk 3]
```diff
@@ -17,7 +17,7 @@
 FILE=example-runner/target/example-runner-*-exec.jar
 # Run the rest example
 if [ -f $FILE ]; then
-  java -Dspring.profiles.active=eureka,rest -jar $FILE
+  java -Dspring.profiles.active=rest,static,debug -jar $FILE
```
Comment (m78233) [inline deployment/local-jvm/example-runner/runLocalJVMExample.sh:20]: I'm reading the list rest, static, debug as a list of profiles.  With that line of thought, I would expect a yaml file for each profile.  I don't see one for debug.

File: deployment/local-jvm/README.md [hunk 4]
```diff
@@ -29,6 +29,13 @@ To run the example locally in JVMs, follow these steps (running commands from th
```
Comment (m78233) [inline deployment/local-jvm/README.md:29]: How about moving the line "To run the example ..." down to before the header Running using the Bash Scripts.

**Review by m78233 — COMMENTED** [review]
Minor suggestions and a question.

File: example-library/src/main/resources/application-example.yaml (hunk 2 above)
Comment (dev930018) [reply example-library/src/main/resources/application-example.yaml:19]: In the application-example we are running a static example on localhost and multiple JVMs, so this specifies where the client should expect the data-service to be located.

**Review by m78233 — APPROVED** [review]

**Review by dbz120896 — APPROVED** [review]

File: deployment/local-jvm/README.md [hunk 5]
```diff
@@ -53,15 +60,16 @@ To run the example locally in JVMs, follow these steps (running commands from th
    >> ls
      drwxrwxrwx audit-service
      drwxrwxrwx data-service
-     drwxrwxrwx discovery-service
      drwxrwxrwx palisade-service
      drwxrwxrwx policy-service
      drwxrwxrwx resource-service
      drwxrwxrwx services-manager
      drwxrwxrwx user-service
```
Comment (pd104923) [inline deployment/local-jvm/README.md:67]: This list needs to be updated with AMS, FRS and TOS

File: deployment/local-jvm/example-runner/verify.sh [hunk 6]
```diff
@@ -33,8 +33,6 @@ if [ -d $DIR ]; then
       cat audit-service-example.log
       echo "*** data-service-example.log"
       cat data-service-example.log
-      echo "*** discovery-service.log"
-      cat discovery-service.log
       echo "*** palisade-service-example.log"
       cat palisade-service-example.log
```
Comment (pd104923) [inline deployment/local-jvm/example-runner/verify.sh:37]: Would be a good idea to add an `echo` and a `cat` line for the AMS, FRS and TOS

File: example-runner/src/main/java/uk/gov/gchq/palisade/example/runner/runner/RestExample.java [hunk 7]
```diff
@@ -56,14 +56,15 @@ private void makeRequest(final String userId, final String fileName, final Strin
...
+        String alice = "Alice";
+        String bob = "Bob";
+        String eve = "Eve";
 
-        final String salary = Purpose.SALARY.name();
-        final String dutyOfCare = Purpose.DUTY_OF_CARE.name();
-        final String staffReport = Purpose.STAFF_REPORT.name();
+        String salary = Purpose.SALARY.name();
+        String dutyOfCare = Purpose.DUTY_OF_CARE.name();
+        String staffReport = Purpose.STAFF_REPORT.name();
```
Comment (pd104923) [inline example-runner/src/main/java/uk/gov/gchq/palisade/example/runner/runner/RestExample.java:67]: Is there a reason why these are no longer `final`?

**Review by pd104923 — CHANGES_REQUESTED** [review]

**Review by pd104923 — APPROVED** [review]

File: example-runner/src/main/java/uk/gov/gchq/palisade/example/runner/runner/RestExample.java (hunk 7 above)
Comment (dev930018) [reply example-runner/src/main/java/uk/gov/gchq/palisade/example/runner/runner/RestExample.java:67]: SonarQube disagrees with Checkstyle over SNAKE_CAPS or camelCase if `final`

**Review by dev-958 — APPROVED** [review]

**Pull request merged**